cmd/kube-proxy/app/server.go

```diff
@@ -434,8 +434,6 @@ type ProxyServer struct {
 	OOMScoreAdj            *int32
 	ResourceContainer      string
 	ConfigSyncPeriod       time.Duration
-	ServiceEventHandler    config.ServiceHandler
```

Comment:
How obnoxious would it be if I asked for this simplification to be a commit of its own?

Reply:
@thockin do you mean just the service event handler distinct from the endpoints? Or something else...

Reply:
I meant the removal of service/enpoint handlers as distinct things.  It's a small but clear refactoring.